src/user.ts

```diff
@@ -471,7 +473,7 @@ export class User {
     );
     const result = await this.userDB.insert(finalNewUser);
     newUser._rev = result.rev;
-    if (this.config.local.sendConfirmEmail) {
+    if (this.config.local.sendConfirmEmail && !this.config.mailer.useCustomMailer) {
```

Comment:
Here is no event emited

Reply:
The event is emitted here:

https://github.com/perfood/couch-auth/blob/325148330d65f4950ba46ec08b637e07eff25f4d/src/user.ts#L421